recipe/meta.yaml

```
@@ -10,6 +10,8 @@ package:
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: da10b034750addbd95a328654d20364c479f4e2e26e0f72933204d61cbc8fa78
+  patch:
```

```suggestion
  patches:
```
sorry my bad, I had a typo in my example.